src/Views/gagepage.html

```diff
@@ -225,13 +225,24 @@ <h4 class="modal-title">StreamStats Gage Page</h4>
                         &nbsp;<i class="fa fa-spinner fa-3x fa-spin loadingSpinner"></i>
                     </div>
                     <center>
-                    <div class="well well-sm" style="background-color:#EBF0F5;" ng-if="vm.chartConfig">
-                        <input type="checkbox" ng-if="vm.formattedFloodFreq.length > 0" id="yesAEPLines" name="yesAEPLines" ng-model="vm.plotlines" ng-change="vm.togglePlotLines()" checked>
-                        <label for="yesAEPLines" ng-if="vm.formattedFloodFreq.length > 0"> AEP Plot Lines&nbsp;&nbsp;</label>
-                        <input type="checkbox" ng-if="vm.formattedDailyFlow.length > 0" id="logScale" name="logScale" ng-model="vm.logScale" ng-change="vm.toggleLogLinear()" checked>
-                        <label for="logScale" ng-if="vm.formattedDailyFlow.length > 0"> Log Scale</label>
-                        <highchart id="chart1" config="vm.chartConfig"></highchart>
-                    </div>
+                        <div class="well well-sm" style="background-color:#EBF0F5;" ng-if="vm.chartConfig">
+                            <div class="plot-filters">
+                                <input type="checkbox" ng-if="vm.formattedFloodFreq.length > 0 || vm.formattedAltFloodFreq.length > 0" id="toggleFloodStats" name="toggleFloodStats" ng-model="vm.showFloodStats" ng-change="vm.toggleFloodStats()" checked>
+                                <label for="toggleFloodStats" ng-if="vm.formattedFloodFreq.length > 0 || vm.formattedAltFloodFreq.length > 0" > Show Flood Statistics:</label>
+                                <select id="gageFloodStatsSelect" ng-if="vm.formattedFloodFreq.length > 0 || vm.formattedAltFloodFreq.length > 0 " style="width:19.25%" name="gageFloodStatsSelect" ng-model="vm.selectedFloodFreqStats" ng-options="floodFreqStat.name for floodFreqStat in vm.allFloodFreqStats" ng-change="vm.chooseFloodStats()" ng-disabled="vm.showFloodStats == false"></select>&nbsp;&nbsp;&nbsp;                      
+                                <input type="checkbox" ng-if="vm.formattedPeakDates.length > 0" id="togglePeaks" name="togglePeaks" ng-model="vm.peaksOnYear" ng-change="vm.togglePeakYear()" checked>
+                                <label for="togglePeaks" ng-if="vm.formattedPeakDates.length > 0"> Plot Peaks on Latest Year&nbsp;&nbsp;</label>
```

Comment:
Recommended text edit to make it more clear what this means

```suggestion
                                <label for="togglePeaks" ng-if="vm.formattedPeakDates.length > 0"> Plot Annual Streamflow Peaks on Latest Year&nbsp;&nbsp;</label>
```